ga4gh/backend.py

```diff
@@ -70,12 +70,13 @@ def __init__(self, request, containerIdMap):
         self._nextObject = None
         self._searchAnchor = None
         self._distanceFromAnchor = None
-        if request.pageToken is None:
+        if request.page_token is None:
             self._initialiseIteration()
         else:
             # Set the search start point and the number of records to skip from
             # the page token.
-            searchAnchor, objectsToSkip = _parsePageToken(request.pageToken, 2)
+            searchAnchor, objectsToSkip = \
```

Comment:
Hanging indent is preferable to splitting with a backslash here. From PEP8:

>The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.

So, here we would have
```python
searchAnchor, objectsToSkip = _parsePageToken(
    request.page_token, 2)
```